images/build/php/docker-php-ext-install.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh
+set -e
```

Comment:
copyright notice

Reply:
these scripts are actually copied from here:
https://github.com/docker-library/php/tree/master/7.3/stretch/apache

Reply:
Let's add a readme.md to this folder and reference it

Reply:
Added a SOURCE.md